package.json

```diff
@@ -1,22 +1,17 @@
 {
-  "name": "govuk-prototype-kit",
-  "description": "Rapidly create HTML prototypes of GOV.UK services",
+  "name": "govuk-prototype-kit-docs",
+  "description": "Site for the GOV.UK Prototype Kit",
   "version": "12.1.1",
   "private": true,
   "engines": {
     "node": ">=12.0.0 <17.0.0"
   },
   "scripts": {
-    "start": "node start",
```

Comment:
isnt it still useful to have a start? To run locally? It's a common node convention I think